truss/templates/control/control/helpers/truss_patch/model_container_patch_applier.py

```diff
@@ -127,6 +141,48 @@ def _apply_system_package_patch(self, system_package_patch: SystemPackagePatch):
         else:
             raise ValueError(f"Unknown python requirement patch action {action}")
 
+    def _apply_config_patch(self, config_patch: ConfigPatch):
+        self._app_logger.debug(f"Applying config patch {config_patch.to_dict()}")
+        TrussConfig.from_dict(config_patch.config).write_to_yaml_file(
+            Path(self._inference_server_home / config_patch.path)
+        )
+
+    def _apply_env_var_patch(self, env_var_patch: EnvVarPatch, inf_env: dict):
+        self._app_logger.debug(
```

Comment:
for a followup (and maybe @squidarth can comment), how should we decide which statements are debug vs info vs warning? I think during runtime, folks will actually want to know what got patch, which suggestions that some of these should be info?

Just opening the convo.